user/tests.py

```diff
@@ -147,3 +147,17 @@ def test_post_blank_password(self):
     self.assertTemplateUsed(self.response, 'user/signup.html')
 
 
+class LoginSuccessTest(TestCase):
+  def setUp(self):
+    self.data = {'username':'people', 'email':'test@test.test', 'password':'testpassword', 'age':'1'}
+    self.test_people = Account.objects.create(username=self.data['username'],email=self.data['email'],password=self.data['password'],age=self.data['age'])
+    self.home_url = reverse('home')
+    self.login_url = reverse('login')
+
+  def test_login_success(self):
+    self.login_data = {'username':'people','password':'testpassword'}
```

Comment:
```suggestion
    login_data = {'username':'people','password':'testpassword'}
```
ここではlogin_dataを他で使うことはないので`self.`の形にする必要はないです